hack/lib/catalogsource.bash

```diff
@@ -161,6 +209,7 @@ function build_image() {
 
 function delete_catalog_source {
   logger.info "Deleting CatalogSource $OPERATOR"
+  oc delete imagecontentsourcepolicy --ignore-not-found=true serverless-image-content-source-policy
```

Comment:
nit: Technically, should this be done after deleting the `catalogsource` ?

Reply:
and perhaps even after deleting the `serverless-index` deployment ?

Reply:
We could move it. But it's not necessary. The ImageContentSourcePolicy is required when the pods try to pull the images. They were already pulled and the pods started. It would be necessary only if the catalog source pod restarted and needed to pull the images again. I can move this in a follow up (unless this PR needs some other changes too).